lib/screens/v2/widget_instance/cr_departures.ex

```diff
@@ -194,4 +195,7 @@ defmodule Screens.V2.WidgetInstance.CRDepartures do
         []
     end
   end
+
+  defp get_track_number(_, "place-forhl"), do: nil
```

Comment:
On the fence about hardcoding vs config. We can leave it for now, and If we get more stations like this in the future, we can re-assess. Let's go ahead and add a comment, though.